src/test/java/com/browserstack/NonPageObjectTest.java

```diff
@@ -0,0 +1,78 @@
+package com.browserstack;
+
+import com.browserstack.utils.ElementLocatorUtil;
+import com.browserstack.utils.JsonUtil;
+import io.qameta.allure.Step;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import static com.browserstack.utils.Constants.Capabilities.CAPABILITY_MASK_COMMAND;
+import static com.browserstack.utils.Constants.Capabilities.CAPABILITY_VALUE_DEFAULT_MASK;
+import static com.browserstack.utils.Constants.ElementLocators.*;
+import static com.browserstack.utils.Constants.ErrorMessages.*;
+
+public abstract class NonPageObjectTest extends AbstractTest {
```

Comment:
rather than calling this class Non{Something} ... can we give a more clearer name w.r.t the objective of the class.
For users who clone our framework code and do not want to use Page objects would like to see this class naming with more clarity on its objective that what it is not.